src/pb_env.cc

```diff
@@ -253,6 +253,19 @@ EnvironmentManager::ExtractIfNotExtracted(std::string env_path)
 
   bool env_extracted = false;
   bool re_extraction = false;
+
+  // If the path is not a conda-packed file, then bypass the extraction process
+  std::size_t subStrPos = env_path.size() > 6 ? env_path.size() - 6 : 0;
+  std::string subPath =  env_path.substr(subStrPos);
+  if (subPath != "tar.gz") {
```

Comment:
Instead of using magic numbers like `6`, can you add a helper like `endsWith(string, suffix)` and use it here? I think it will be clearer. 

I believe PB already has boost dependencies installed, so you can probably just use:
```cpp
if (!boost::algorithm::ends_with(env_path, "tar.gz")) {
  ...
}
```
Otherwise lots of simple implementations here: https://www.techiedelight.com/check-if-a-string-ends-with-another-string-in-cpp/
```
if (!EndsWith(env_path, "tar.gz")) {
  ...
}
```